README.md

```diff
@@ -0,0 +1,23 @@
+# URL Shortener
+
+### https://url-shortener-dere-tara.herokuapp.com/
+
```

Comment:
deploying to heroku is cool.
was there any resource or tutorial you followed? (if yes send us the link please)

Reply:
@rajtandel21 https://youtu.be/Li0Abz-KT78